install/kubernetes/helm/subcharts/mixer/templates/deployment.yaml

```diff
@@ -267,6 +303,14 @@
         - name: istio-certs
           mountPath: /etc/certs
           readOnly: true
+        {{- if $.Values.global.sds.enabled }}
+        - name: sds-uds-path
+          mountPath: /var/run/sds
+        {{- if $.Values.global.sds.useTrustworthyJwt }}
```

Comment:
What happened with the non-trusted jwt ? 

Also if possible add an option for no JWT ( spire )

Reply:
k8s api server will mount non-trustworty jwt by default, so no token mount is needed. 
we could discuss non-jwt case later.